app/services/import_actuals.rb

```diff
@@ -176,6 +198,10 @@ def convert_value(value)
       raise I18n.t("importer.errors.actual.non_numeric_value")
     end
 
+    def convert_refund_value(value)
```

Comment:
When we have converter methods that need to exist for the sake of metaprogramming, we tend to define just the one method and alias the other to it (example: https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/blob/develop/app/services/activities/import_from_csv.rb#L392)

Reply:
:+1: